brainstormforce/ultimate-addons-for-gutenberg
# Normal/Hover tab missing for info box.

### Description

- Normal/Hover tab missing for Icon/Image and CTA type Text/Button.

### Screenshots

- output - https://share.getcloudapp.com/2NuP5Ood

### Types of changes




- New feature (non-breaking change which adds functionality)

### How has this been tested?

- Drag and drop Info-box.
- Go to Style open Icon/Image tab.
- Enable CTA type Text/Button and check CTA tab in Style.

### Checklist:
